ios/RCTWKWebView/RCTWKWebViewManager.m

```diff
@@ -27,21 +27,42 @@ @interface RCTWKWebViewManager () <RCTWKWebViewDelegate>
 
 @end
 
+@implementation RCTConvert (WKWebViewConfiguration)
+
++ (WKWebViewConfiguration *)WKWebViewConfiguration:(id)json
+{
+  NSDictionary<NSString *, id> *options = [RCTConvert NSDictionary:json];
+  WKWebViewConfiguration *config = [[WKWebViewConfiguration alloc] init];
+  config.allowsInlineMediaPlayback = [RCTConvert BOOL:options[@"allowsInlineMediaPlayback"]];
+  config.mediaPlaybackRequiresUserAction = [RCTConvert BOOL:options[@"mediaPlaybackRequiresUserAction"]];;
```

Comment:
Oh.  Wait.  That's only available since iOS 10.

Probably it is simpler to just have this call to the deprecated method for now, rather than switching on the SDK version here.